Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "win_pe_analysis"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+pe-parser = "0.6.0"
```

Comment:
Are you happy with the capabilities of this library? From the available PE parser options do you think this is the most solid one?

Reply:
There is another crate (library) called exe-rs which is built for parsing and building PE files. It seemed to me when I was first looking through them that, since we don't need the building aspect, that this library sufficed. The other library does seem to have a couple niceties (built in hashing, possibly more robust testing before deployment) that I missed on my first look that may mean I try that one out. My main concern when I originally looked is I didn't want to add bloat to the dependencies and it seemed like the library I chose might have a faster runtime.

In fact, I just spent a decent time looking and it looks like I missed a bunch. I'll probably switch to that crate. It should also make dealing with malformed PE's easier. I should be able to get a PR/commit out later today or some time on Monday

Reply:
If you want to continue with this crate for now, and refactor for another library later is fine with me.

 we still need to process PEFiles to generate a first dataser and train an initial model, although a bad one.

Theres plenty of time to use a different one, but I think you should pick your parser wisely beca later this will restrain th project quite a bit down the line.